main.star

```diff
@@ -200,3 +218,70 @@ def start_btp_relayer(plan, src_chain, dst_chain, config_data):
     btp_bridge.start_relayer(plan, src_chain, dst_chain, config_data, src_btp_address, dst_btp_address, config_data["bridge"])
 
     return config_data
+
+def generate_config_data(args):
+
+    data = get_args_data(args)
+    config_data = {
+        "links": data.links,
+        "chains" : {
+            "%s" % data.src : {},
+            "%s" % data.dst : {}
+        },
+        "contracts" : {
+            "%s" % data.src : {},
+            "%s" % data.dst : {}
+        },
+        "bridge" : data.bridge
+    }
+
+    return config_data
+
+def get_args_data(args):
+
+    links = args["links"]
+    source_chain = links["src"]
+    destination_chain = links["dst"]
+
+    if destination_chain == "cosmwasm" and source_chain == "cosmwasm":
+        destination_chain = "cosmwasm1"
+
+    bridge = args["bridge"]
+
+    return struct(
+        links = links,
+        src = source_chain,
+        dst = destination_chain,
```

Comment:
@Prathiksha-Nataraja This is redundant code . I feel you can make use for the package_io input_parser.star

Reply:
resloved